setup.py

```diff
@@ -144,7 +144,8 @@ def find_packages(path, base=""):
     "authlib>=0.10",
     "pyopenssl",
     "dnspython>=1.12.0",
-    "zeroconf-monkey>=0.1.0"
+    "zeroconf-monkey>=0.1.0",
+    "PyJWT"
```

Comment:
You don't need this since you are using authlib. Check out https://docs.authlib.org/en/latest/jose/index.html

Reply:
Cheers! And thanks even more for changing the license on authlib :smile: